src/app/shared/models/notification-handler.ts

```diff
@@ -8,9 +8,10 @@ export enum NotificationType {
 }
 
 export enum NotificationActionType {
-  ALL,
+  SCHEDULE_ALL,
   TEST,
-  CANCEL
+  CANCEL_ALL,
+  CANCEL_SINGLE
```

Comment:
should we add `SCHEDULE_SINGLE` functionality too? For the clinical tasks and others in the future

Reply:
Ok I've added it to the notification handler but I'll add the functionality in a separate PR since this would require more changes to the scheduling as well.